acms/acms.yml

```
@@ -49,11 +49,13 @@ starter_kits:
         - acquia_cms_search:^1.3.5
         - acquia_cms_tour:^1.3.0
         - acquia_cms_toolbar:^1.3.3
+        - consumer_image_styles:^4.0.7
```

@lauriii I think we can remove the version restrictions. Let composer download the latest stable release (unless we want to forcefully restrict to some version release). We've plan to remove this strict version dependencies for all other acquia_cms modules as well.
```suggestion
        - consumer_image_styles
```

---

Is there downside to restricting the major version? That would ensure we are not pulling modules incompatible with the basic-starter

---

Hi @lauriii No, there's no downside I see on restricting to major version. Only thought I had was, let composer choose & download the latest stable release. If team feels ok with it, then we are good as well. (For integrated any new major release, this needs to be updated back :) ).